### ISSUE TITLE
Customize Module

### ISSUE THREAD

**Issue by huangwentao1996** [HudsonAlpha/aRNAPipe#2]
Hi,

I was trying to customize the modules which used by aRNAPipe. For example, I want to disable the gatk module. But the documents does not provide a instructions for customize the module. When I delete the STAR module in config.txt, a error message wil appear. Like the following:
Traceback (most recent call last):
  File "/home/adigioia/Documents/Matt/program/aRNAPipe/lib/wr_aRNApipe.py", line 68, in <module>
    print "  - STAR:            " + var["star"]
KeyError: 'star'

Could you tell me how to customize the modules used by aRNApipe?

**Comment by arnald-alonso** [comment]
Hi,

I've just realized that the wiki doesn't include specific information on how to disable/enable modules.  I've updated it in '2. aRNApipe: The main application > The config.txt file'. In order to disable a module the user has to set the first argument ('a') of the corresponding module to 0. All the customization is done through the configuration file by modifying the arguments but avoiding line deletions.
